index.js

```diff
@@ -3,10 +3,22 @@ const path = require('path');
 const { getSelectedDbs } = require('./utils');
 const selected = getSelectedDbs();
 
-const makePath = (type) => path.resolve(__dirname, `dbs/GeoLite2-${type}.mmdb`);
+const makePath = (edition) => path.resolve(__dirname, `dbs/${edition}.mmdb`);
 
 const paths = selected.reduce((a, c) => {
-  a[c.toLowerCase()] = makePath(c);
+  const aliases = {
+    'geolite2-asn': 'asn',
+    'geolite2-city': 'city',
+    'geolite2-country': 'country',
+  }
+  // The keys are lower-case edition IDs
+  key = c.toLowerCase()
```

Comment:
Now that things a getting a bit more complicated, it'd be good to use actual db names as keys. I'm mainly concerned about users of the lib having to "translate" db names into path keys which isn't great for usability.

It was straightforward with 3 free options, but with a bunch of paid ones it's not looking great.

Let's keep the old ones for backwards compatibility and say they're aliases:

city => GeoLite2-City
country => GeoLite2-Country
asn => GeoLite2-ASN

and stop advertising the old keys in the readme.